src/test/suite/fixtures/recorded/surroundingPair/parseTree/json/clearRound2.yml

```diff
@@ -0,0 +1,23 @@
+languageId: json
+command:
+  version: 0
+  spokenForm: clear round
+  action: clearAndSetSelection
+  targets:
+    - type: primitive
+      modifier: {type: surroundingPair, delimiter: parentheses}
+initialState:
+  documentContents: // (hello)
```

Comment:
Note that we support comments in json now because we fallback to text-based if we're in a parse-tree error node